Makefile

```diff
@@ -49,12 +88,16 @@ $(shell grep $(1) $(GLOOE_DIR)/build_env | cut -d '=' -f 2-)
 endef
 
 .PHONY: build-plugins
+verify-plugin: build-plugins
```

Comment:
this needs a .PHONY verify-plugin, and the .PHONY build-plugins should be moved to the target below

Reply:
Ive renamed the target so the .PONY build-plugin compiles and verifies the plugin.
But I cannot get the verify binary to work and keep getting:
```
chmod +x _glooe/verify-plugins-linux-amd64
_glooe/verify-plugins-linux-amd64 -pluginDir plugins -manifest plugins/plugin_manifest.yaml
make: _glooe/verify-plugins-linux-amd64: Command not found
make: *** [Makefile:101: verify-plugin] Error 127
```
So the file exists, but it cannot be executed somehow.